saleor/graphql/schema.graphql

```diff
@@ -256,6 +272,42 @@ input AppInput {
   permissions: [PermissionEnum]
 }
 
+type AppInstall {
+  errors: [Error!]! @deprecated(reason: "Use typed errors with error codes. This field will be removed after 2020-07-31.")
+  appErrors: [AppError!]!
+  appJob: AppOngoingInstallation
+}
+
+input AppInstallInput {
+  appName: String
+  manifestUrl: String
+  activateAfterInstallation: Boolean = true
+  permissions: [PermissionEnum]
+}
+
+enum AppJobStatus {
+  PENDING
+  SUCCESS
+  FAILED
+  DELETED
+}
+
+type AppOngoingInstallation {
```

Comment:
I would suggest removing the word "ongoing" as this is actually determined by the job status. Just `AppInstallation` here and in queries and mutations would be short and clear.

Reply:
Done, Sir! 👮‍♂️